tests/python/pytorch/graphbolt/test_feature_fetcher.py

```diff
@@ -0,0 +1,41 @@
+import dgl
+import dgl.graphbolt
+import pytest
+import torch
+
+
+def get_graphbolt_fetch_func():
+    feature_store = dgl.graphbolt.feature_store.InMemoryFeatureStore(
```

Comment:
https://github.com/dmlc/dgl/pull/5914 is renaming it, @czkkkkkk 
whoever merge later, please make sure fix according it.